Website Overhaul

Beissner opened the pull request:
Here's a start for the new website, still a lot to do but wanted to get this pushed and get some feedback. Since we'll be (hopefully) adding more routes in the near future, I didn't change anything inside `about` but I did re-organize the `home` folder. If the structure doesn't make sense let me know and I'll redo that. 

I followed Amber's sketch mockup, but there's a lot of blank space and I still need to add in the row icons we talked about. I think it also may need a splash of color and I'd be grateful if Ashley could weigh in on that.

Hopefully from here we can create smaller issues so other folks can jump in and contribute.

Beissner commented:
I apologize for my commit history! Promise next PR will be better.

ambergkim commented on src/components/header/header.scss:
```diff
@@ -1,62 +0,0 @@
-
```
🙌 Desktop is looking good in Chrome. Will just later need small adjustments for other browsers like Safari (for example, the shadows are not adjusting the gradient).

Read up on Mobile-first development and media queries. Once we've implemented it, I think we'll be ok to put this up online. Basically, we want to develop for the smallest smartphone screens first and then grow from there, applying media queries for bigger and bigger screens on top of the smaller screen sizes. Use the browser dev-tools to preview what it'll look like on the different screen sizes.
<img width="1103" alt="Screen Shot 2020-05-19 at 9 51 39 AM" src="https://user-images.githubusercontent.com/14047327/82355640-573e7f80-99b7-11ea-99ae-aadaa7b89afc.png">


Check out this line. Previous to this, the queries above it will support small screens. This is a breakpoint where we start to support a screen that's a bit bigger. https://github.com/nourimeals/website/blob/4e8def8ea3f7422b4a4d4e883f6755e02c2f5c96/src/components/home/home.scss#L51

RaevLogic commented:
It's definitely a good start, keep it up.

Beissner commented:
Added in a media query for small phones and tablets `max-width: 767px`. For those smaller devices I moved the "donate" button out of the header into the main body to allow for more space. Let me know how it looks for you.

ambergkim commented on src/components/home/home.scss:
```diff
@@ -1,294 +1,95 @@
-@media only screen and (min-width: 700px) {
+/*      MEDIA QUERIES     */
+/* small phones to small tablets: from 481px to 767px */
+@media only screen and (max-width: 767px) {
```
With mobile-first development, min-width is typically used rather than max-width. SSo the organization/logic would be backwards in nesting and might look like this for scss

`
# Smallest up to the first breakpoint. Let's say, 700px
section {
     flex-direction: column;
     
     # screen sizes starting from 700px and above
     @media only screen and (min-width: 700px) {
            min-height: 100vh;
            width: 100%;
            display: flex;
      }
}
`

Beissner commented:
Just pushed the media query changes that reflect mobile first design. It's looking better but still needs small fixes like adding the favicon, using `rem` to adjust font size and adjusting the board images. Thinking we could create issues for these smaller items?

ambergkim commented:
@Beissner This is good. Good job! The last thing is the mobile nav/hamburger menu and we'll be good to merge it in. We can definitely make small issues for the adjustments so more folks can contribute and the tasks are more bite sized.
